.clang-tidy

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
For which clang-tidy version is this for?

Reply:
It's not solved, that's the problem :( same with clang-format where versions are _slightly_ incompatible (that's why we fix a specific version in osrm for all devs).

Reply:
Overall @danpat is using `node-cpp-skel` latest here, so I'll share context below on the decisions in node-cpp-skel. If you want to change/improve/debate `node-cpp-skel` design decisions feel free to open tickets over at https://github.com/mapbox/node-cpp-skel/issues.

Specifically node-cpp-skel framework installs its own version of clang-tidy/clang-format as a way to provide a consistent version for devs (when you type `make`). If you want to change the clang-tidy version (and are using the framework from node-cpp-skel like @danpat is doing here) you now control the version via the `mason-versions.ini` file.

Reply:
👍 You can either use `make tidy` or `./scripts/clang-tidy.sh` to run the project-preferred version of `clang-tidy`

If someone wants to use a system version of `clang-tidy`, well, there's not much we can do to stop them.  Our CI builds will only use the invocations above though, so ultimately, CI is our enforcer on this.